react/README.md

```diff
@@ -1,36 +1,50 @@
-# Boxwise react client 
+# Readme
 
-This front-end project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This frontend project of Boxtribute was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
I think in our brand style guide we agreed that we should use "Front-end" and "back-end" rather than frontend and backend. We should agree and be consistent with this, so let's do a find-and-replace, or make me a good case for why we should update the messaging guidelines.